packages/radix-ui-themes/src/components/card.css

```diff
@@ -115,148 +58,123 @@
   --card-border-width: 1px;
-  background-color: var(--color-panel);
+  --card-background-color: var(--color-panel);
 
+  transition: box-shadow 120ms;
+  box-shadow: var(--base-card-classic-box-shadow-outer);
+
+  &::before {
+    background-color: var(--card-background-color);
+  }
   &::after {
-    border-radius: var(--card-after-border-radius);
-    /* A transparent shadow to transition smoothly to `:active` */
-    box-shadow: 0 0 0 1px var(--color-transparent), var(--shadow-2);
-    transition: box-shadow 120ms;
+    box-shadow: var(--base-card-classic-box-shadow-inner);
   }
-
   &:where(button, [href]) {
     @media (hover: hover) {
       &:where(:hover) {
+        transition-duration: 40ms;
+        box-shadow: var(--base-card-classic-hover-box-shadow-outer);
         &::after {
-          transition-duration: 40ms;
-          box-shadow: var(--card-classic-hover-box-shadow);
+          box-shadow: var(--base-card-classic-hover-box-shadow-inner);
         }
       }
     }
     &:where([data-state='open']) {
+      transition-duration: 40ms;
+      box-shadow: var(--base-card-classic-hover-box-shadow-outer);
       &::after {
-        transition-duration: 40ms;
-        box-shadow: var(--card-classic-hover-box-shadow);
+        box-shadow: var(--base-card-classic-hover-box-shadow-inner);
       }
     }
     &:where(:active:not([data-state='open'])) {
+      transition-duration: 40ms;
+      box-shadow: var(--base-card-classic-active-box-shadow-outer);
       &::after {
-        transition-duration: 40ms;
-        box-shadow: 0 0 0 1px var(--gray-a5), var(--shadow-2);
-      }
-      &:where(:focus-visible) {
-        background-color: var(--gray-a3);
+        box-shadow: var(--base-card-classic-active-box-shadow-inner);
       }
     }
   }
 }
 
-/* ghost */
+/* * * * * * * * * * * * * * * * * * * */
+/*                                     */
+/*          Variants / Ghost           */
+/*                                     */
+/* * * * * * * * * * * * * * * * * * * */
 
 .rt-Card:where(.rt-variant-ghost) {
   --card-border-width: 0px;
 
+  /* We reset the defined margin variables to avoid inheriting them from a higher component */
+  /* If a margin IS defined on the component itself, the utility class will win and reset it */
+  --margin-top: 0px;
+  --margin-right: 0px;
+  --margin-bottom: 0px;
+  --margin-left: 0px;
+
+  /* Define the overrides to incorporate the negative margins */
+  --margin-top-override: calc(var(--margin-top) - var(--card-padding));
+  --margin-right-override: calc(var(--margin-right) - var(--card-padding));
+  --margin-bottom-override: calc(var(--margin-bottom) - var(--card-padding));
+  --margin-left-override: calc(var(--margin-left) - var(--card-padding));
+
+  margin-top: var(--margin-top-override);
+  margin-right: var(--margin-right-override);
+  margin-bottom: var(--margin-bottom-override);
+  margin-left: var(--margin-left-override);
+
+  /* Reset the overrides */
+  .rt-CardInner {
```

Comment:
`rt-CardInner` doesn't exist anymore right as it's one element.
How do we ensure the margin overrides are reset for children?

Reply:
Oh right, good spot. Updated to `:where(&) > *` like we do in other instances